### PR TITLE
#177 Integrate Crashlytics for crash reporting

### PR DESCRIPTION
**Pull request by tuancoltech**

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -80,6 +80,10 @@
             android:name=".synchronization.AppListActivity"
             android:configChanges="orientation|screenSize"></activity>
 
+        <meta-data
+            android:name="io.fabric.ApiKey"
+            android:value="aa8a780391c17fadbc4d60a380447bbcac9ac150" />
```
Comment (sladomic) [inline app/src/main/AndroidManifest.xml]: @jo-elimuai shouldn't we try to keep such keys secret?

Comment (tuancoltech) [reply app/src/main/AndroidManifest.xml]: That's actually my personal Fabric API key.
I'll think about hiding it.

**Comment by sladomic** [comment]
@tuancoltech sorry I wasn't finished testing. Can you see any crash logs from my device in the dashboard?

**Comment by tuancoltech** [comment]
@sladomic I see ai.elimu.appstore_custom.pilot package added, but no crash log for this package.

**Comment by sladomic** [comment]
@tuancoltech ok, just tested it and it worked

**Comment by tuancoltech** [comment]
Should I merge the change now @sladomic ?

**Comment by sladomic** [comment]
@jo-elimuai do you have any objections?

**Comment by jo-elimuai** [comment]
I'll register an account that we can use.

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (jo-elimuai) [reply app/src/main/AndroidManifest.xml]: The key should be hidden, yes. We could add the API key only for "release" builds. I'm working on a solution for this now.

**Pull request merged**